Gems/PhysX/Code/Tests/PhysXTestCommon.h

```diff
@@ -48,6 +48,9 @@ namespace PhysX
 
         void ResetPhysXSystem();
 
+        // Physics Math Helpers
```

Comment:
suggestion: A clearer comment for this function might be useful as opposed to a section comment (e.g. how to use it, what it does)

Reply:
Noted in internal backlog ticket and removing from this PR since this code is not used for the remaining test for the focus of this PR.

GHIs will be created this week.